src/com/google/javascript/jscomp/AbstractCommandLineRunner.java

```diff
@@ -1275,8 +1283,9 @@ void outputJsonStream() throws IOException {
     }
   }
 
-  private void outputModuleBinaryAndSourceMaps(List<JSModule> modules, B options)
-      throws IOException {
+  private DiagnosticType outputModuleBinaryAndSourceMaps(
```

Comment:
I think it would be easier to read if this method returns an int (0 or 1), and just reports the error instead of returning it.